app/src/main/java/com/nextcloud/talk/conversationlist/ConversationsListActivity.kt

```diff
@@ -2221,5 +2233,6 @@ class ConversationsListActivity :
         private const val SIXTEEN_HOURS_IN_SECONDS: Long = 57600
         const val LONG_1000: Long = 1000
         private const val NOTE_TO_SELF_SHORTCUT_ID = "NOTE_TO_SELF_SHORTCUT_ID"
+        private const val CONVERSATION_ITEM_HEIGHT = 44
```

Comment:
not the nicest solution but fine for me as the handling will be replaced with Compose anyway